### PR TITLE
setup jest and webpack

### PR DESCRIPTION
**Pull request by Mahamadou-Nouridine**
Hey @mosidrum , In this Pr I've done these tasks:

- Setup webpack;
- Setup Jest;
- Create the Readme.md file;
- Setup files structure;
- Setup linters

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```
Comment (mosidrum) [inline webpack.config.js:33]: Well done brother. Good Job, I am suggesting we will need a rule to load images from our assest folder

**Review by mosidrum — CHANGES_REQUESTED** [review]
well-done-barack-obama


Well done bro, I think there is single rule we are missing on our webpack configure file

**Comment by Mahamadou-Nouridine** [comment]
Thank you for the review. But since we are using image URLs provided by a CDN rather than images files, we don't need an image loader in the Webpack configuration

**Comment by mosidrum** [comment]
approved-498-x-278-gif-5n3qgmxj2i2btqgc
## APROVED.. :ok_hand:
Let's move to the next task

**Pull request merged**